cmd/image-builder/README.md

```diff
@@ -186,9 +186,9 @@ Image Builder supports three build backends:
 
 - kaniko
 - BuildKit
-- Azure DevOps pipelines
+- ADO pipelines
 
-kaniko and BuildKit build images locally while the Azure DevOps pipelines backend call ADO API.
+kaniko and BuildKit build images locally while the ADO pipelines backend call ADO API.
```

Comment:
```suggestion
kaniko and BuildKit build images locally, while the ADO pipelines backend call ADO API.
```
I'd use a comma here, and also I don't understand this sentence: "the ADO pipelines backend call ADO API". Who calls? The pipelines? Or the backend?

Reply:
The Image Builder client calls ADO API when using ADO pipelines backend. The call to the ADO API is used to start ADO pipeline, where we execute the real image build process.

Maybe this one would be better?

kaniko and BuildKit build images locally, while for the ADO pipelines backend, Image Builder calls ADO API to start build process.